installation/single_machine_docker.rst

```diff
@@ -16,18 +16,18 @@ This section describes setting up a Citus cluster on a single machine using dock
 
 .. code-block:: bash
 
-  curl -sSL https://get.docker.com/ | sudo bash
-  sudo usermod -aG docker $USER && newgrp docker
+  curl -sSL https://get.docker.com/ | sh
```

Comment:
That scary fake error isn't good. Looks like it may be related to running Ubuntu on Amazon hardware specifically. The fix may be running `sudo modprob aufs` but I don't think they'll be updating the install script to do it. https://github.com/docker/docker/issues/14658